src/graphql/client.ts

```diff
@@ -35,7 +35,7 @@ export const apolloClient = new ApolloClient({
       },
     } as TypedTypePolicies,
   }),
-  link,
+  link: (link as unknown) as ApolloLink,
```

Comment:
why does not fit? can you share the error?

Reply:
I downgraded types from 17.0.1 to 17.0.0, it looks like it works without type casting, so I removed it